lib/service-manager.js

```diff
@@ -4,8 +4,11 @@ var configDefaults = require('./config').configDefaults;
 var debug = require('debug')('strong-pm:service-manager');
 var util = require('util');
 
-function ServiceManager(server) {
+function ServiceManager(server, enableTracing) {
   this.server = server;
+  this.enableTracing = enableTracing;
+  if (enableTracing)
+    configDefaults.start = 'sl-run --cluster=cpu --trace';
```

Comment:
@kraman @Setogit shouldn't this be something like `configDefaults.start += ' --trace';`, to not assume that --cluster=cpu is current start command?